rcore/flash.c

```diff
@@ -260,6 +283,30 @@ uint32_t flash_get_resource_address(uint16_t slot_id)
             return i;
         }
     }
-    
+
     return 0;
 }
+
+const File fs_find_file(const char *name)
+{
```

Comment:
make sure to respect `_fs_valid` -- we haven't finished populating our internal state otherwise